packages/store/src/action.ts

```diff
@@ -13,7 +13,7 @@ export function Action(actions: any | any[], options?: ActionOptions) {
     }
 
     for (const action of actions) {
-      const type = action.type;
+      const type: string = action.type;
```

Comment:
Do we really need this? Can't it imply that?

Reply:
Yes, I think it became inferrable once I improved the major typings. Will review.